stardis/io/base.py

```diff
@@ -86,4 +84,14 @@ def parse_config_to_model(config_fname):
         continuum_interaction_species=[],
     )
 
+    # if (
```

Comment:
What's with this big comment block?

Reply:
The big comment block is to allow for rescaled mass fractions (which was the original purpose of this PR. After discussion, we decided that I should use tardis composition and then add that functionality to it, rather than expanding stardis composition functionality. I wanted to keep this code around and ready to uncomment (or at least be a starting point) for after I implement the rescaling into tardis composition. This also applies to the config expansion I did, which is currently unused.